src/utils/get-package-json.mjs

```diff
@@ -0,0 +1,16 @@
+import { readFile } from 'fs/promises'
+import { dirname, join } from 'path'
+import { fileURLToPath } from 'url'
+
+let packageJson
+
+const getPackageJson = async () => {
+  if (!packageJson) {
+    const packageJsonPath = join(dirname(fileURLToPath(import.meta.url)), '../../package.json')
+    packageJson = JSON.parse(await readFile(packageJsonPath))
+  }
+
+  return packageJson
```

Comment:
This now also caches the root `package.json` of the cli as I do not think it will change during runtime.